components/carousel/carousel.md

```diff
@@ -0,0 +1,36 @@
+Carousel:
+
+    <div>
+      <Carousel
+        images={[
+          'http://lorempixel.com/600/400/city',
```

Comment:
is it possible to use custom tag with text or something like that? or can I add custom attributes to img tag?